game.py

```diff
@@ -0,0 +1,32 @@
+import random
+while(True):
+    n=random.randint(1,100)
+    i=1
+    while(True):
+        h=0
+        while(True):
+            h=input("hads bezan : ")
+            if h.isdigit():
+                h=int(h)
+                break
+            else:
+                print("dobare vard kon : ")
+        i+=1
+        if i==6:

```

Comment:
بهتره که ۶ رو constant تعریف کنی

زیر ایمپورت ها 

```python
MAX_TRY = 6
```
و اینجا هم طبق اون استایل گاید بنویس
```suggestion
        i += 1
        if i == MAX_TRY:

```